react/typings/global.d.ts

```diff
@@ -235,4 +237,11 @@ declare global {
     startLoading: () => void
     stopLoading: () => void
   }
+
+  interface FormattedBlock {
+    id: Extension['blockId']
+    role: 'AFTER' | 'AROUND' | 'BEFORE' | 'BLOCK' | 'TEMPLATE'
```

Comment:
Why uppercase?

Reply:
The `role` is declared as an enum in the GraphQL schema, and that is one of the ways to represent their values in JS/TS. I believe it's a good option because, apart from the single quotes, it looks almost the same as the actual enum values (`AFTER`, `AROUND`, etc).